src/main/java/org/folio/inventory/dataimport/handlers/actions/UpdateItemEventHandler.java

```diff
@@ -44,8 +45,12 @@
 
   private static final Logger LOG = LoggerFactory.getLogger(UpdateItemEventHandler.class);
 
+  public static final String ERROR_MSG_KEY = "ERROR_MSG";
+  public static final String FAILED_EVENT_KEY = "FAILED_EVENT";
   private static final String PAYLOAD_HAS_NO_DATA_MSG = "Failed to handle event payload, cause event payload context does not contain MARC_BIBLIOGRAPHIC data or ITEM to update";
+  private static final String STATUS_UPDATE_ERROR_MSG = "Could not change item status '%s' to '%s'";
   private static final String ITEM_PATH_FIELD = "item";
+  private static final String[] protectedStatusesFromUpdate = new String[]{"Aged to lost", "Awaiting delivery", "Awaiting pickup", "Checked out", "Claimed returned", "Declared lost", "Paged", "Recently returned"};
```

Comment:
Is it possible to use `ItemStatus` enum values?

Reply:
do you mean one of the existing enums?

Reply:
Yes, there is an enum with all the possible statuses, just think if we can use enum constants instead of raw string.

Reply:
got it, but here we only need some statuses which are disallowed being updated. The existing enum is used for subsequent status validation after mapping.